core/templates/dev/head/pages/base.html

```diff
@@ -1,5 +1,5 @@
 {% macro warnings_and_loader() -%}
-  <div ng-cloak>
+  <div id="blk-content" ng-cloak>
```

Comment:
Instead of an href tag for designating the anchor link destination, I added a data attribute to the skip link button. As shown below:
 
![screen shot 2017-03-09 at 2 05 30 am](https://cloud.githubusercontent.com/assets/9513121/23828968/725c664c-068f-11e7-8259-0d6d051b58a7.png)

That data attribute is the id of that blk-content tag, which is used to control the page scroll focus here in utilsServices:

![screen shot 2017-03-11 at 7 15 47 pm](https://cloud.githubusercontent.com/assets/9513121/23828978/a85de2ca-068f-11e7-9dff-135306cd25f5.png)

Initially there wasn't an id in that div to target for the link or an encompassing tag for the content reproduced . Perhaps there is a better way to implement this solution though?

Or I could just select the "oppia-toast-container" right beneath that ng-cloak div?

![screen shot 2017-03-11 at 7 22 30 pm](https://cloud.githubusercontent.com/assets/9513121/23829001/347b040e-0690-11e7-91b2-31e84c6e0c9a.png)